test_requirements.txt

```diff
@@ -10,7 +10,8 @@ pytest>=3.0,<3.3; python_version<"3.4"
 coverage==4.4.2
 flake8==3.5.0
 psycopg2==2.7.3
-pylint==1.7.4
+pylint==1.8.1; python_version>'3.3'
+pylint==1.7.5; python_version=='3.3'
```

Comment:
What's the reason for this change?

Reply:
Python 1.8 dropped support for Python 3.3.